diff --git a/base/BUILD.gn b/base/BUILD.gn
index 0c24c7f29c342b36f5a0de76ac44c578ad285ce1..3a0f4ad5d6a2b97089c78b716a65784845cd22b7 100644
--- a/base/BUILD.gn
+++ b/base/BUILD.gn
@@ -4574,6 +4574,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/ValueChangedCallback.java",
       "android/java/src/org/chromium/base/WrappedClassLoader.java",
     ]
+    sources += brave_java_base_sources
 
     if (use_clang_profiling) {
       sources += [
@@ -5112,6 +5113,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/shared_preferences/SharedPreferencesManager.java",
       "android/java/src/org/chromium/base/shared_preferences/StrictPreferenceKeyChecker.java",
     ]
+    sources += brave_base_shared_preferences_java_sources
   }
 
   android_library("flagged_apis_java") {
