diff --git a/chrome/browser/about_flags.cc b/chrome/browser/about_flags.cc
index 4f920b1932858f1b1e13616752229affcd9ab468..cb7f339fd69bcf9d11f2763eaab05aa466f8edfe 100644
--- a/chrome/browser/about_flags.cc
+++ b/chrome/browser/about_flags.cc
@@ -4452,6 +4452,7 @@ const FeatureEntry::FeatureVariation kEnableCanvasNoiseVariations[] = {
 //
 // When adding a new choice, add it to the end of the list.
 const FeatureEntry kFeatureEntries[] = {
+    BRAVE_ABOUT_FLAGS_FEATURE_ENTRIES
 // Include generated flags for flag unexpiry; see //docs/flag_expiry.md and
 // //tools/flags/generate_unexpire_flags.py.
 #include "build/chromeos_buildflags.h"
