diff --git a/chrome/browser/ui/android/toolbar/java/src/org/chromium/chrome/browser/toolbar/top/ToolbarTablet.java b/chrome/browser/ui/android/toolbar/java/src/org/chromium/chrome/browser/toolbar/top/ToolbarTablet.java
index 44f09d4b7425b6cc3de99f0651a7f6e810a8beb6..cf17ca2108d456edcfbad071c26e85593d99c59e 100644
--- a/chrome/browser/ui/android/toolbar/java/src/org/chromium/chrome/browser/toolbar/top/ToolbarTablet.java
+++ b/chrome/browser/ui/android/toolbar/java/src/org/chromium/chrome/browser/toolbar/top/ToolbarTablet.java
@@ -327,10 +327,12 @@ public class ToolbarTablet extends ToolbarLayout
                     getContext(), getToolbarDataProvider().getTab(), /* fromAppMenu= */ false);
             RecordUserAction.record("MobileToolbarDownloadPage");
         }
+        BraveToolbarLayout.class.cast(this).onClickImpl(v);
     }
 
     @Override
     public boolean onLongClick(View v) {
+        if (BraveToolbarLayout.class.cast(this).onLongClickImpl(v)) return true;
         String description = null;
         Context context = getContext();
         Resources resources = context.getResources();
@@ -451,6 +453,7 @@ public class ToolbarTablet extends ToolbarLayout
         mLocationBar.getTabletCoordinator().tintBackground(textBoxColor);
         mLocationBar.updateVisualsForState();
         setToolbarHairlineColor(color);
+        super.onThemeColorChanged(color, shouldAnimate);
 
         // Notify the StatusBarColorController of the toolbar color change. This is to match the
         // status bar's color with the toolbar color when the tab strip is hidden on a tablet.
