diff --git a/chrome/browser/ui/webui/side_panel/bookmarks/bookmarks.mojom b/chrome/browser/ui/webui/side_panel/bookmarks/bookmarks.mojom
index ad12ee3470305b451b5c0a29fd757658e7180f33..df2b595b6eb7f0491b16e8cc23b66ca31551d87c 100644
--- a/chrome/browser/ui/webui/side_panel/bookmarks/bookmarks.mojom
+++ b/chrome/browser/ui/webui/side_panel/bookmarks/bookmarks.mojom
@@ -23,9 +23,10 @@ enum SortOrder {
   kLastOpened = 2,
   kAlphabetical = 3,
   kReverseAlphabetical = 4,
+  kCustom = 5,
 
   // Must be last. May add new values before this one.
-  kCount = 5,
+  kCount = 6,
 };
 
 // Used to indicate the order bookmarks are sorted in the side panel.
