diff --git a/content/browser/BUILD.gn b/content/browser/BUILD.gn
index 399896c77046e2e71fe9553a263ffcb1882dad89..21bad6da3edb063ec061f508eda99573ce126a61 100644
--- a/content/browser/BUILD.gn
+++ b/content/browser/BUILD.gn
@@ -3686,6 +3686,7 @@ source_set("browser") {
   if (is_chromeos) {
     deps += [ "//ash/constants:constants" ]
   }
+  import("//brave/content/browser/sources.gni") sources += brave_content_browser_sources deps += brave_content_browser_deps
 }
 
 if (is_android) {
