diff --git a/third_party/tflite/BUILD.gn b/third_party/tflite/BUILD.gn
index de54a29e8ac95fefb9655ae3fd057a35168f5f99..dcd52ee2d97ece3ca1b774689563d0e9ca7939f1 100644
--- a/third_party/tflite/BUILD.gn
+++ b/third_party/tflite/BUILD.gn
@@ -219,6 +219,7 @@ tflite_static_library("tflite_builtin_op_resolver") {
     "//services/webnn/*",
     "//third_party/mediapipe/*",
   ]
+  import("//brave/third_party/tflite/tflite.gni") visibility += brave_tflite_builtin_op_resolver_visibility
 
   conditional_deps = [ ":tflite" ]
 }
