diff --git a/src/codegen/compiler.cc b/src/codegen/compiler.cc
index 7de13fe37475f2963c1b8a3c3949b8dd51973fdf..f5427766bbefaa5a4cd7f7c94009281f88d1a71d 100644
--- a/src/codegen/compiler.cc
+++ b/src/codegen/compiler.cc
@@ -3377,6 +3377,7 @@ MaybeDirectHandle<JSFunction> Compiler::GetFunctionFromEval(
     }
   }
   CHECK(is_compiled_scope.is_compiled());
+  BRAVE_COMPILER_GET_FUNCTION_FROM_EVAL
 
   return result;
 }
